### PR TITLE
[Documentation] Sylius Plus docs part 2; including fixes to links

### PR DESCRIPTION
**Pull request by CoderMaggie**
| Q               | A
| --------------- | -----
| Branch?         | 1.6
| Doc fix?        | yes - broken links for The Book
| New docs?    | yes
| Related tickets | Part 2 of #11054 
| License         | MIT

### CODE REVIEW

File: docs/book/architecture/emails.rst [hunk 1]
```diff
@@ -74,6 +74,99 @@ You have the following parameters available:
...
+**Code**: ``sylius_plus_return_request_resolution_changed``
+
+**The default template**:
+``@SyliusPlusPlugin/Returns/Infrastructure``
+``/Resources/views/Emails/returnRequestResolutionChangedNotification.html.twig``
+
+Parameters:
+
+* ``returnRequest`` which has been rejected
+* ``order`` of the rejected return request
```
Comment (GSadee) [inline docs/book/architecture/emails.rst]: Sth like:
```suggestion
* ``returnRequest`` whose resolution has been changed
* ``order`` of the modified return request
```

File: docs/book/architecture/emails.rst [hunk 2]
```diff
@@ -74,6 +74,99 @@ You have the following parameters available:
...
+**Code**: ``sylius_plus_return_request_repaired_items_sent``
+
+**The default template**:
+``@SyliusPlusPlugin/Returns/Infrastructure``
+``/Resources/views/Emails/returnRequestRepairedItemsSentNotification.html.twig``
+
+Parameters:
+
+* ``returnRequest`` which has been rejected
+* ``order`` of the rejected return request
```
Comment (GSadee) [inline docs/book/architecture/emails.rst]: Should be also adjusted to the current email

File: docs/book/architecture/emails.rst [hunk 3]
```diff
@@ -74,6 +74,99 @@ You have the following parameters available:
...
+
+Return Request: Repaired Items Sent
+'''''''''''''''''''''''''''''''''''
+
+This email is sent when the administrator marks that a return request's repaired items have been sent back to the Customer.
```
Comment (GSadee) [inline docs/book/architecture/emails.rst]:
```suggestion
This email is sent when the administrator marks that return request's repaired items have been sent back to the Customer.
```
?

File: docs/book/customers/map.rst.inc [hunk 4]
```diff
@@ -1,8 +1,4 @@
-.. raw:: html
-
-    <ul class="simple">
-    <li><p><a class="reference internal" href="customers/customer_and_shopuser.html"><span class="doc">Customer and ShopUser</span></a></p></li>
-    <li><p><a class="reference internal plus-feature" href="customers/customer_pools.html"><span class="doc">Customer Pools</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/adminuser.html"><span class="doc">AdminUser</span></a></p></li>
-    <li><p><a class="reference internal" href="customers/addresses/index.html"><span class="doc">Addresses</span></a></p></li>
-    </ul>
\ No newline at end of file
+* :doc:`/book/customers/customer_and_shopuser`
```
Comment (GSadee) [inline docs/book/customers/map.rst.inc]:
```suggestion
* :doc:`/book/customers/customer_and_shop_user`
```
? 😄

File: docs/book/customers/map.rst.inc [hunk 5]
```diff
@@ -1,8 +1,4 @@
...
+* :doc:`/book/customers/customer_pools`
+* :doc:`/book/customers/adminuser`
```
Comment (GSadee) [inline docs/book/customers/map.rst.inc]:
```suggestion
* :doc:`/book/customers/admin_user`
```

File: docs/book/orders/map.rst.inc [hunk 6]
```diff
@@ -1,16 +1,12 @@
...
-    <li><p><a class="reference internal" href="orders/coupons.html"><span class="doc">Coupons</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/payments.html"><span class="doc">Payments</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/invoices.html"><span class="doc">Invoices</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/shipments.html"><span class="doc">Shipments</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/checkout.html"><span class="doc">Checkout</span></a></p></li>
-    <li><p><a class="reference internal plus-feature" href="orders/returns.html"><span class="doc">Returns</span></a></p></li>
-    <li><p><a class="reference internal" href="orders/refunds.html"><span class="doc">Refunds</span></a></p></li>
-    </ul>
+* :doc:`/book/orders/orders`
+* :doc:`/book/orders/cart-flow`
```
Comment (GSadee) [inline docs/book/orders/map.rst.inc]: So what convention do we have? `-` or `_`?

File: docs/book/customers/map.rst.inc (hunk 4 above)
Comment (CoderMaggie) [reply docs/book/customers/map.rst.inc]: the file is named like that already, can be changed but ina different PR.

File: docs/book/customers/map.rst.inc (hunk 5 above)
Comment (CoderMaggie) [reply docs/book/customers/map.rst.inc]: sorry, it's the filename, I cannot change it in the scope of this PR.

File: docs/book/orders/map.rst.inc (hunk 6 above)
Comment (CoderMaggie) [reply docs/book/orders/map.rst.inc]: `_` in general, but there are some mistakes in the filenames made previously.

File: docs/book/architecture/emails.rst (hunk 3 above)
Comment (CoderMaggie) [reply docs/book/architecture/emails.rst]: why?

**Comment by GSadee** [comment]
Thanks, Magdalena! :1st_place_medal:

**Pull request merged**